src/PeopleForm.tsx

```diff
@@ -0,0 +1,297 @@
+import React, {
+  BaseSyntheticEvent, useState, useMemo, useCallback
+} from 'react';
+import { PeopleFormProps } from './typesDefinitions';
+import { NAME_REGEXP, CURRENT_YEAR, FILTER_PARENTS } from './constAndFunc';
+import classNames from 'classnames';
```

Comment:
```suggestion
import cn from 'classnames';
```
Так как это дефолтный экспорт, ты можешь называть функцию как угодно